openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OParamUtils.java

```diff
@@ -1,39 +1,28 @@
 /*
- * Copyright 2018 Feedzai
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * The copyright of this file belongs to Feedzai. The file cannot be
+ * reproduced in whole or in part, stored in a retrieval system,
+ * transmitted in any form, or by any means electronic, mechanical,
+ * photocopying, or otherwise, without the prior permission of the owner.
  *
+ * © 2019 Feedzai, Strictly Confidential
```

Comment:
Please fix the copyright messages (in this and new files) to be what we had before (with Apache license and copyright feedzai). You can configure copyrights per project.